backend/tsconfi.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
+    "resolveJsonModule": true,
```

Comment:
We already have a `tsconfig.json` in our backend folder - and this is also a `tsconfi.json` without a `g` so it won't be used anyway.